drivers/wifi/winc1500/Kconfig.winc1500

```diff
@@ -15,6 +15,12 @@ config WIFI_WINC1500_NAME
 	string "Driver name"
 	default "WINC1500"
 
+config WINC1500_DRV_USE_OLD_SPI_SW
+	bool "Driver to use Old SPI software"
+	default n
```

Comment:
The option value is `n` by default, so this line can be omitted.

Reply:
Sure.

Reply:
Done.